src/ReaderScreen/reader.jsx

```diff
@@ -29,14 +30,20 @@ function Reader({ navigation, route }) {
   const [isHeader, toggleIsHeader] = useState(true);
   const [rowHeights, setRowHeights] = useState([]);
   const [itemsCount] = useState(50);
+  const { title } = route.params.params;
 
   const [isLayout, toggleLayout] = useState(false);
   const dispatch = useDispatch();
   const { shabad, isLoading } = useFetchShabad(shabadID);
   const { currentPage, fetchScrollData } = usePagination(shabad, itemsCount);
   useSaveScroll(isLayout, currentPage, readerRef, currentScrollPosition, shabadID);
   useBookmarks(readerRef, shabad, bookmarkPosition, rowHeights, layoutHeight);
+  const nightModeColors = nightColors(isNightMode);
+  const { backgroundColor } = nightModeColors;
 
+  const { safeAreaViewBack } = nightModeColors;
```

Comment:
If we are not using nightModeColors anywhere else, I think we can directly get the backgroundColor and safeAreaViewBack.

`const { backgroundColor, safeAreaViewBack }= nightColors(isNightMode);`

Reply:
fixed